src/GraphQl/Mutation/MutationResolverInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\GraphQl\Mutation;
+
+use GraphQL\Type\Definition\ResolveInfo;
+
+/**
+ * @experimental
+ *
+ * @author Raoul Clais <raoul.clais@gmail.com>
+ */
+interface MutationResolverInterface
+{
+    /**
+     * @return mixed The mutated item
+     */
+    public function __invoke($item, array $input, ResolveInfo $info);
```

Comment:
```suggestion
    public function __invoke($item, array $context);
```
Maybe having a context is better to be independent of webonyx/graphql-php.
Then in context we have:
```php
['input' => [], 'resolve_info' => $info]
```